docs/iris/src/sphinxext/generate_logo.py

```diff
@@ -0,0 +1,441 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+
+"""
+Script to generate the Iris logo in every required format.
+Uses XML ElementTree for SVG file editing.
+"""
+
+from collections import OrderedDict, namedtuple
+from copy import deepcopy
+from io import BytesIO
+from os import environ
+from pathlib import Path
+from re import sub as re_sub
+from xml.etree import ElementTree as ET
+from xml.dom import minidom
+
+from cartopy import crs as ccrs, __version__ as cartopy_version
+from cartopy.feature import LAND
+from matplotlib import pyplot as plt, rcParams
+from numpy import linspace
```

Comment:
Use standard import form.
```suggestion
import numpy as np
```

Reply:
Thanks, good to learn that's an important distinction to the community.